packages/angular-output-target/src/output-angular.ts

```diff
@@ -68,23 +68,27 @@ export function generateProxies(
   const dtsFilePath = path.join(rootDir, distTypesDir, GENERATED_DTS);
   const componentsTypeFile = relativeImport(outputTarget.directivesProxyFile, dtsFilePath, '.d.ts');
   const includeSingleComponentAngularModules = outputTarget.includeSingleComponentAngularModules ?? false;
+  const includeOutputImports = components.some((component) => component.hasEvent);
```

Comment:
This will need to be updated to filter out internal events. We only require creating the proxy for the event if it is public.

e.g.: https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/angular-output-target/src/output-angular.ts#L149

Context from Stencil: https://github.com/ionic-team/stencil/blob/9690a8226424db4bf2da4540bfdf826980c916ac/src/compiler/transformers/component-build-conditionals.ts#L28C1-L30

Reply:
@sean-perkins thanks for pointing this out! 
I applied changes - please kindly check in a spare moment.